src/map/instance.cpp

```diff
@@ -257,3 +259,130 @@ bool CInstance::CheckFirstEntry(uint32 id)
     //insert returns a pair (iterator,inserted)
     return m_enteredChars.insert(id).second;
 }
+
+void CInstance::DisengageAll()
```

Comment:
i don't really understand the usefulness of this - how many instances are going to use this? an instance isn't necessarily a bcnm - was anything preventing you from doing it in script?